Document changes v6.2.0 #429 (#430)
enonic/app-guillotine: docs/clients.adoc

image::images/graphiql-ide.png[Query playground interface]

> Starting from the version 6.2.0 GraphiQL was updated to GraphiQL 2, that was replaced with new UI.

Reviewer: ```suggestion
Starting from the version 6.2.0 Guillotine was updated to use GraphiQL 2 with a new enhanced UI.
```

* The left panel is used to display documentation generated from the GraphQL API
* The center panel allows you to edit your GraphQL query
* The right panel displays the result of the query execution

> * The top right corner is extended by the button group for choosing the branch

Reviewer: ```suggestion
* Branch selector (Draft/Master) can be found in the top right corner
```
